pipeline.py

```diff
@@ -11,7 +11,18 @@
 import oneaday_filter
 import result_formatter
 import scraper_connection
-from petrarch import petrarch
+#from petrarch2 import petrarch2
+
+server_details, file_details, petrarch_version = utilities.parse_config('PHOX_config.ini')
+if petrarch_version == '1':
+    from petrarch import petrarch
+    print("Using original Petrarch version")
```

Comment:
Probably want to change these to `logger.info()` statements instead. `print()` should generally be status info that let's people know that the program didn't die. `logger.info` should be information about the config and runtime. I settled on this since the `print` can be ephemeral but `logger` gets written to disk.